edb/server/pgcluster.py

```diff
@@ -1240,6 +1241,15 @@ async def _log_subprocess_output(
     return []
 
 
+async def _safe_readline(stream: asyncio.StreamReader) -> bytes:
+    try:
+        line = await stream.readline()
+    except ValueError:
```

Comment:
Does stream actually recover from this error?

Reply:
Yes